lib/src/colorpicker.dart

```diff
@@ -290,18 +294,28 @@ class _ColorPickerState extends State<ColorPicker> {
               mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
There's some padding just above this line that looks like this:
`padding: const EdgeInsets.fromLTRB(15.0, 5.0, 10.0, 5.0),`

When `widget.displayColorIndicator` is false, could you make it so the left and right padding are both `10`? Else we end up with the slider being pushed slightly to the right.